core/src/com/cloud/storage/template/HttpTemplateDownloader.java

```diff
@@ -117,7 +116,7 @@ public boolean retryMethod(final HttpMethod method, final IOException exception,
             request = new GetMethod(downloadUrl);
             request.getParams().setParameter(HttpMethodParams.RETRY_HANDLER, myretryhandler);
             completionCallback = callback;
-            //this.request.setFollowRedirects(false);
```

Comment:
The change was originally done on our 4.7 branch before being ported to master and then here. Even back in 4.7 this line was commented.